### ISSUE TITLE
Azure DevOps-Change Azure AD Connection-Prerequisites-Global Administrator Permissions-Question

### ISSUE THREAD

**Issue by jonmikeli** [MicrosoftDocs/vsts-docs#4971]
Hi,

I have been working on changing Azure AD of a few Azure DevOps and there is something not clear to me in the provided [documentation](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/change-azure-ad-connection?view=azure-devops).

Does the Microsoft account used for the operation need to have Global Administrator permissions in the source Azure AD **AND** the target Azure AD or **ONLY** in the source Azure AD?
I would say only in the source Azure AD but having had issues in the process, I am wondering if this could have had an impact.

Thanks.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 98f0caaf-2697-c506-339b-2dea64233e48
* Version Independent ID: 7cd2ea02-2079-85d0-09e0-a90af59937d9
* Content: [Change connection to Azure AD - Azure DevOps Services](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/change-azure-ad-connection?view=azure-devops)
* Content Source: [docs/organizations/accounts/change-azure-ad-connection.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/organizations/accounts/change-azure-ad-connection.md)
* Product: **devops**
* Technology: **devops-accounts**
* GitHub Login: @chcomley
* Microsoft Alias: **chcomley**

**Comment by chcomley** [comment]
Hi @jonmikeli, Thanks for your feedback. You only need to be a _member_ in Azure AD. You must be a **Project Collection Administrator in your Azure DevOps organization**. I've published an update to the doc, clarifying this, which should go live sometime this afternoon. If you have any technical difficulties, please don't hesitate to create a support ticket here: https://visualstudio.microsoft.com/vs/support/#talktous
Thanks again!

**Comment by jonmikeli** [comment]
Hi @chcomley,

Thanks for your fast answer and the update. Perfect.

Just curiosity: was the Global Administrator permission required at some point in the past and now, it is not required anymore thanks to changes/improvements? Or was it a documentation issue since the very beginning?

Thanks for the link to create support tickets.
We already created one last Friday and everything was fixed within a few hours. I just wanted to be sure about the right procedure before running other Azure AD connection changes in the company (and avoid issues :)).
Amazing job from the Support Team by the way.

Congrats to both of you. Very efficient :).
